methods.py

```diff
@@ -13,6 +14,14 @@ def get_elements(self, locator):
         """Returns all elements for the specific locator"""
         return self.driver.find_elements(*locator)
 
+    def get_elements_by_text(self, locator):
+        """Returns element with specific text for the specific locator"""
```

Comment:
please add the type of `locator` variable  in the docstring or in parameters of the function, like
`get_elements_by_text(self, locator: tuple)` or maybe `get_elements_by_text(self, locator: list)`